### PR TITLE
Networking for Docker Sibling containers

### PR DESCRIPTION
**Pull request by temyers**

### CODE REVIEW

**Comment by Hamo69** [comment]
Is there any more @temyers ?

**Comment by temyers** [comment]
Nothing unless @williamsia has any comments to make.

**Comment by Hamo69** [comment]
👍

File: _posts/2017-09-27-networking_Sibling_Docker_Containers.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+</head>
+<body>
+  Hello World!
+</body>
+</html>
+```
+
+BOOM! 🤘
+
+[Docker networking](https://docs.docker.com/engine/userguide/networking/)is used by `docker-compose` under the hood to transparently add all your compose services to a common network.
```
Comment (williamsia) [inline _posts/2017-09-27-networking_Sibling_Docker_Containers.markdown]: put a space before the "is"

**Pull request merged**